lib/cache/cache.go

```diff
@@ -68,6 +69,7 @@ func ForProxy(cfg Config) Config {
 		{Kind: services.KindAuthServer},
 		{Kind: services.KindReverseTunnel},
 		{Kind: services.KindTunnelConnection},
+		{Kind: services.KindRemoteCluster},
```

Comment:
Have you tested this with older clusters? I ran into a problem when I was adding support for `services.AppServer` in Application Access. See now I fixed it in this commit:

https://github.com/gravitational/teleport/pull/4582/commits/0700990077d59fce75f91bcad140c8de1e438851

Reply:
I will merge my PR after you merge application access to get a magic fix :)